pkg/api/types.go

```diff
@@ -1479,6 +1479,18 @@ type NodeStatus struct {
 	DaemonEndpoints NodeDaemonEndpoints `json:"daemonEndpoints,omitempty"`
 	// Set of ids/uuids to uniquely identify the node.
 	NodeInfo NodeSystemInfo `json:"nodeInfo,omitempty"`
+	// List of container images on this node
+	Images []ContainerImage `json:"images",omitempty`
+}
+
+// Describe a container image
+type ContainerImage struct {
+	// ID of the image.
+	ID string
```

Comment:
I added it just in case we may need some identifier, but seems not like that if narrowed down to this PR/issue, should I remove it?